rs/bitcoin/adapter/src/lib.rs

```diff
@@ -193,16 +193,31 @@ impl AdapterState {
 }
 
 /// Starts the gRPC server and the router for handling incoming requests.
-pub fn start_grpc_server_and_router(
-    config: &config::Config,
+pub fn start_server(
```

Comment:
Do you want me to switch to this function in my PocketIC PR?

Reply:
exactly